crm_salesperson_planner/models/crm_salesperson_planner_visit_template.py

```diff
@@ -68,14 +69,31 @@ class CrmSalespersonPlannerVisitTemplate(models.Model):
         string="Number of Sales Person Visits", compute="_compute_visit_ids_count"
     )
     auto_validate = fields.Boolean(default=True)
-    rrule_type = fields.Selection(
-        default="daily",
-        required=True,
-    )
     last_visit_date = fields.Date(compute="_compute_last_visit_date", store=True)
     final_date = fields.Date(string="Repeat Until")
     allday = fields.Boolean(default=True)
-    recurrency = fields.Boolean(default=True)
+    """Set all compute=_compute_recurrence fields of calendar.event as store=True.
+    We want to manage the value of the fields manually and we don't want to depend
+    on recurrence_id field (only possible with calendar.event).
+    We don't use the recurrency field either because it is unnecessary."""
```

Comment:
This should be comments, not docstring:

```suggestion
    # Set all compute=_compute_recurrence fields of calendar.event as store=True.
    # We want to manage the value of the fields manually and we don't want to depend
    # on recurrence_id field (only possible with calendar.event).
    # We don't use the recurrency field either because it is unnecessary.
```